src/routes/jira/server/jira-server-url-post.ts

```diff
@@ -3,11 +3,64 @@ import axios from "axios";
 import { GitHubServerApp } from "models/github-server-app";
 import { isValidUrl } from "utils/is-valid-url";
 
+interface MessageAndCode {
+	errorCode: string;
+	message: string;
+	statusCode: number;
+}
+
+interface GheServerUrlErrorResponses {
+	[key: string | number]: MessageAndCode;
+}
+
+interface GheServerUrlErrors {
+	codeOrStatus: GheServerUrlErrorResponses
+}
+
+const gheServerUrlErrors: GheServerUrlErrors = {
+	codeOrStatus: {
+		invalidUrl: {
+			errorCode: "GHE_ERROR_1",
+			message: "Invalid URL",
+			statusCode: 200
```

Comment:
Sending a 200 response for invalidUrl, ENOTFOUND, and default as none of these are server errors and don't receive a an error status code in the response. Sending `200` with success `false`